datalad/metadata/aggregate.py

```diff
@@ -698,12 +705,14 @@ def __call__(
                 agginfo_db,
                 to_save)
             # update complete
-            yield get_status_dict(
+            res = get_status_dict(
                 status='ok',
                 action='aggregate_metadata',
                 path=parentds_path,
                 type='dataset',
                 logger=lgr)
+            res.update(agginfo_db.get(parentds_path, {}))
```

Comment:
FWIW we have `datalad.utils.updated` so could have been
```python
yield updated(
    get_status_dict(
￼                  status='ok',
￼                  action='aggregate_metadata',
￼                  path=parentds_path,
￼                  type='dataset',
￼                  logger=lgr),
    agginfo_db.get(parentds_path, {})
)
```
not sure if makes it anyhow better here (would be more difficult to debug if smth happens) but here you go